### PR TITLE
1.0: Add tech preview disclaimer

### PR DESCRIPTION
**Pull request by jgu17**
Added tech preview disclaimer on the welcome page.

(cherry picked from commit 9e0ccd75e325e150ab606c0c7f71093e7ca9c1eb)

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
-and lifecycle management on SUSE Container as a Service Platform (CaaSP) and
-SUSE Enterprise Storage (SES), using Airship, shell scripts, and Ansible playbooks.
+The socok8s project automates SUSE Containerized OpenStack provisioning and
+lifecycle management on SUSE Container as a Service Platform (CaaSP) and
+SUSE Enterprise Storage (SES), using Airship, OpenStack Helm, shell scripts,
+and Ansible playbooks.
+
+As a technology preview, the included features are not supported. They may be
+functionally incomplete, unstable or in other ways not suitable for production
+use. They are mainly included for customer convenience and give customers a
```
Comment (dirkmueller) [inline doc/source/index.rst]: s,customers,users,

File: doc/source/index.rst [hunk 2]
```diff
@@ -1,9 +1,33 @@
...
+
+  * Keystone
+  * Glance
+  * Cinder
+  * Nova
+  * Neutron
+  * Heat
+  * Horizon
+
+Whether it will be moved to a fully supported product later depends on customer
```
Comment (dirkmueller) [inline doc/source/index.rst]: s,on customer,on your,

File: doc/source/index.rst [hunk 3]
```diff
@@ -1,9 +1,33 @@
...
+and market feedback. A technology preview does not automatically result in
+support at a later point in time. It could be dropped at any time and SUSE is not
+committed to providing a technology preview later in the product cycle.
+
+Please, give your SUSE representative feedback, including your experience and
+use case.
```
Comment (dirkmueller) [inline doc/source/index.rst]:
```suggestion
relevance to your use case.
```

File: doc/source/index.rst [hunk 4]
```diff
@@ -1,9 +1,33 @@
...
+  * Glance
+  * Cinder
+  * Nova
+  * Neutron
+  * Heat
+  * Horizon
+
+Whether it will be moved to a fully supported product later depends on customer
+and market feedback. A technology preview does not automatically result in
+support at a later point in time. It could be dropped at any time and SUSE is not
```
Comment (dirkmueller) [inline doc/source/index.rst]:
```suggestion
support at a later point in time. It is provided under the terms of the Apache-2.0 license and SUSE is not
```

File: doc/source/index.rst [hunk 5]
```diff
@@ -1,9 +1,33 @@
...
+committed to providing a technology preview later in the product cycle.
```
Comment (dirkmueller) [inline doc/source/index.rst]:
```suggestion
comitting to providing a technology preview later in the product cycle.
```

File: doc/source/index.rst (hunk 2 above)
Comment (jgu17) [reply doc/source/index.rst]: done

File: doc/source/index.rst (hunk 1 above)
Comment (jgu17) [reply doc/source/index.rst]: done

File: doc/source/index.rst (hunk 5 above)
Comment (jgu17) [reply doc/source/index.rst]: I think the right grammar is "committed"

**Comment by gmingsuse** [comment]
Looks good to me.

**Comment by JanZerebecki** [comment]
This is not a backport. master currently has currently after it was reverted in https://github.com/SUSE-Cloud/socok8s/pull/554 no disclaimer. Is this intentional?

**Comment by jgu17** [comment]
> 
> 
> This is not a backport. master currently has after it was reverted in #554 no disclaimer. Is this intentional?
Yes intentionally revert the change on master as it doesn't need the tech preview disclaimer. Unintentionally, this pr still said backport.

**Pull request merged**